From 84c822da9bf6cb6e8f0a52dc95cdd01d7b604a36 Mon Sep 17 00:00:00 2001
From: Matthias <matgnt@gmail.com>
Date: Tue, 20 Apr 2010 19:08:54 +0200
Subject: [PATCH] Disable qmlviewer qml.

Because of the error:
bin/qml: hidden symbol `NetworkAccessManagerFactory::setupProxy(QNetworkAccessManager*)' isn't defined
in v4.7.0-tp1
---
 tools/tools.pro |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/tools.pro b/tools/tools.pro
index 3ed07b1..34b0aa1 100644
--- a/tools/tools.pro
+++ b/tools/tools.pro
@@ -26,7 +26,7 @@ mac {
 
 embedded:SUBDIRS += kmap2qmap
 
-contains(QT_CONFIG, declarative):SUBDIRS += qml
+#contains(QT_CONFIG, declarative):SUBDIRS += qml
 contains(QT_CONFIG, dbus):SUBDIRS += qdbus
 !wince*:contains(QT_CONFIG, xmlpatterns): SUBDIRS += xmlpatterns xmlpatternsvalidator
 embedded: SUBDIRS += makeqpf
-- 
1.6.3.3

